change/@griffel-core-1e5e67a5-96c2-4f20-9d02-feb94abc63ae.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Support sourcemap for griffel devtools",
```

```suggestion
  "type": "patch",
  "comment": "feat: Add support for source maps in devtools",
```

nit: if there will be no bump - we will not get a version with needed features 🥹